taxi/views.py

```diff
@@ -80,10 +110,27 @@ class CarDeleteView(LoginRequiredMixin, generic.DeleteView):
 
 
 class DriverListView(LoginRequiredMixin, generic.ListView):
-    model = Driver
+    model = get_user_model()
     paginate_by = 5
 
 
 class DriverDetailView(LoginRequiredMixin, generic.DetailView):
-    model = Driver
+    model = get_user_model()
     queryset = Driver.objects.all().prefetch_related("cars__manufacturer")
+
```

Comment:
While the 'model' attribute is set to 'get_user_model()', indicating a change in the expected model for the view, it's crucial to note that the 'queryset' attribute directly references 'Driver.objects.all()' without incorporating the model derived from 'get_user_model()'. As a result, the 'queryset' doesn't align with the updated 'model'.